apps/voting-disputable/test/helpers/voting.js

```diff
@@ -1,14 +1,43 @@
 const { bigExp } = require('@aragon/apps-agreement/test/helpers/lib/numbers')
+const { encodeCallScript } = require('@aragon/contract-test-helpers/evmScript')
+const { decodeEventsOfType } = require('@aragon/apps-agreement/test/helpers/lib/decodeEvent')
 
-const pct = x => bigExp(x, 16)
+const EMPTY_SCRIPT = '0x00000001'
 
-const getVoteState = async (voting, id) => {
-  const isOpen = await voting.isVoteOpen(id)
-  const { executed, startDate, snapshotBlock, supportRequired, minAcceptQuorum, voteOverruleWindow, earlyExecution, yea, nay, votingPower, script } = await voting.getVote(id)
-  return { isOpen, isExecuted: executed, startDate, snapshotBlock, support: supportRequired, quorum: minAcceptQuorum, overruleWindow: voteOverruleWindow, earlyExecution, yeas: yea, nays: nay, votingPower, script }
-}
+module.exports = (web3, artifacts) => {
```

Comment:
I wonder if we should avoid this pattern of injecting the truffle environment transparently, so it's more obvious how to both use the module and how the module is used in a test.

For example:

- It's not immediately clear which imports require this injection
- It can lead to some awkward situations like

```js
const { pct } = require('../helpers/voting')()
```

It would be more clear if we only surfaced direct exports from a module, especially as you can't do the `require('module')()` trick when importing ES modules. For helpers that specifically need the environment, we could migrate them to a `helpers/web3/` folder or etc. and expose a default export from there to hydrate all the internals that assume the environment to exist.

Reply:
Totally agreed! This one was definitely a workaround, based on the new proposed changes on https://github.com/aragon/contract-helpers/pull/35 we should be able to fix this soon